harfbuzz-sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "harfbuzz-sys"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
Why bump version before a release prep? (We aren’t ready for a release.)

Reply:
I'm bumping the version here so that Servo can use this new feature. What's left to be done before a release?

Reply:
Update to a less than 3 year old version of Harfbuzz.

Reply:
(To be fair, the version used in this repo is now only a bit over a year old.)